server/views/pages/bookAVisit/visitors.njk

```diff
@@ -62,52 +63,85 @@
 
       <h1 class="govuk-heading-l">{{ pageHeaderTitle }}</h1>
 
-      <p><strong>Prisoner name:</strong> <span data-test="prisoner-name">{{ prisonerName }}</span></p>
+      <h2 class="govuk-heading-m" data-test="prisoner-name">{{ prisonerName }} active restrictions and alerts</h2>
 
-      {% if restrictions | length %}
-      <div class="prisoner-restrictions">
-        <h2 class="govuk-heading-m">Prisoner's active restrictions</h2>
+      <p>Review restrictions and alerts before selecting visitors.</p>
+
+      <p>This page shows alerts that are relevant for social visits. You can also <a class="govuk-link--no-visited-state" href="{{ prisonerDpsAlertsUrl }}" target="_blank" data-test="all-alerts-link">view all alerts</a>.</p>
+
+      {% if restrictions | length or activeAlerts | length %}
         {% set prisonerRestrictions = [] %}
         {% for restriction in restrictions %}
-          {% set restrictionStartDate %}{% if restriction.startDate %}{{ restriction.startDate | formatDate }}{% else %}Not entered{% endif %}{% endset %}
-          {% set restrictionExpiryDate %}{% if restriction.expiryDate %}{{ restriction.expiryDate | formatDate }}{% else %}Not entered{% endif %}{% endset %}
+          {% set restrictionExpiryDate %}{% if restriction.expiryDate %}{{ restriction.expiryDate | formatDate }}{% else %}No end date{% endif %}{% endset %}
           {%- set prisonerRestrictions = (prisonerRestrictions.push([
             {
-              html: '<span class="test-restrictions-type' + loop.index + ' govuk-tag restriction-tag restriction-tag--' + restriction.restrictionType + '">' + restriction.restrictionTypeDescription + '</span>'
+              html: '<span data-test="test-restrictions-type' + loop.index + '" class="govuk-tag restriction-tag restriction-tag--' + restriction.restrictionType + '">' + restriction.restrictionTypeDescription + '</span>'
```

Comment:
The `test-` part of the attribute value is redundant: e.g. rather than `data-test="test-restrictions-type"`, just have `data-test="restrictions-type"` (a few others like this too)